cli/tests/test_projects.rs

```diff
@@ -0,0 +1,42 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+// Copyright 2023 Oxide Computer Company
+
+use assert_cmd::Command;
+use httpmock::MockServer;
+use oxide_api::types::{Project, ProjectResultsPage};
+use oxide_httpmock::MockServerExt;
+use predicates::prelude::*;
+
+/// Validate `oxide api` for a simple GET with parameters.
+#[test]
+fn text_xxx() {
+    let server = MockServer::start();
+
+    let mock = server.project_list(|when, then| {
+        when.limit(10.try_into().unwrap());
+        then.ok(&ProjectResultsPage {
+            items: vec![Project::builder()
+                .name("proj")
+                .description("dummy description")
+                .id(uuid::Uuid::new_v4())
+                .try_into()
+                .unwrap()],
+            next_page: None,
+        });
+    });
```

Comment:
cool. how would you share a given mock across multiple tests?

Reply:
You could throw in the inputs or outputs into a `const`